src/constants/tokens.ts

```diff
@@ -264,6 +282,70 @@ export const WETH_POLYGON = new Token(
   'WETH',
   'Wrapped Ether'
 )
+export const CELO_CELO = new Token(
+  SupportedChainId.CELO,
+  '0x471EcE3750Da237f93B8E339c536989b8978a438',
+  18,
+  'CELO',
+  'Celo native asset'
```

Comment:
Again, why include an ERC20 for the native asset, instead of just using the native asset?